login_password_page.php

```diff
@@ -81,8 +81,13 @@
 
 # Get the user id and based on the user decide whether to continue with native password credential
 # page or one provided by a plugin.
+#
+# OR
+#
+# if the plugin provides autoprovisioning, than we can also redirect to the plugin page
```

Comment:
Please explain the expected flow(s) when user is not found.  Explain the case where use should be provisioned and the other where user wouldn't be provisioned because it doesn't exist in identity system.

Reply:
In my opinion, user provisioning must always  run afterwards authentication. If I use a plugin for auth and the user not yet exists in the system, than I want to do provisioning only, if the user authenticated himself correctly (that's why I set the flag for autoprov capable)

The flow:
- if user found -> go to plugin page and to the auth
- if user not found:
  - if plugin can do autoprov, than continue to auth
  - if pluging cannot do autoprov, than skip plugin auth and use standard flow